base/constants.go

```diff
@@ -46,7 +46,7 @@ const SASLPasswordKey = "saslPassword"
 const HttpGet = "GET"
 
 // default values for configurable parameters if not specified by user
-const BucketOpTimeout uint64 = 120
+const BucketOpTimeout uint64 = 20
```

Comment:
I hope this is a expected change and not causing: "did not get all vb uuid. len(vbuuidMap) =0"

Reply:
It was expected - the 120 seconds timeout was too long

Reply:
The retry mechanism doesn't work correctly even with the 20 second change - will need to revisit later